admin_guide/image_policy.adoc

```diff
@@ -108,12 +108,48 @@ The ImageStream "bar" is invalid:
 
 [[image-policy-configuring-the-image-policy-admission-plug-in]]
 == Configuring the ImagePolicy Admission Plug-in
+To configure which images can run on your cluster, configure the ImagePolicy
+Admission plug-in in the `*_master-config.yaml_*` file. You can set one or more
+rules as required. Following are examples of some of the rules you can set:
 
-To enable this feature, configure the plug-in in `*_master-config.yaml_*`:
+* *Reject images with a particular annotation*:
++
+Use this rule to reject all images that have a specific annotation set on them.
+For example, to reject all images that have the annotation
+`images.openshift.io/deny-execution` create the following rule:
++
+[source, yaml]
+----
+- name: execution-denied
+  onResources:
+  - resource: pods
+  - resource: builds
+  reject: true
+  matchImageAnnotations:
+  - key: images.openshift.io/deny-execution <1>
+    value: "true"
+  skipOnResolutionFailure: true
+----
+<1> This annotation may be set by infrastructure that wishes to flag particular images as dangerous.
```

Comment:
I think the footnote above would serve better in the introduction on line 117? So:

"If a particular annotation has been deemed harmful, you can set a rule to reject all images that use a specific annotation. The following rejects all images using the `images.openshift.io/deny-execution` annotation"

Or something like that?

Reply:
`If a particular annotation has been deemed harmful,` I mean to say: `If a particular image has been deemed harmful administrators can set this annotation to flag those images.`